### ISSUE TITLE
Separate "Sort" into "Sort" and "Filter"

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#373]
Right now the "Sort by" drop down for the projects page includes both soting & filtering options.

For example, if I select "sort by verified" I ONLY see verified priojects... so it's actually "filtering by verified"

Would love it if we could have separate menus for "sort by" and "filter by", so it would say "category, sort by, filter by, search" from left to right.

![image.png](https://images.zenhubusercontent.com/5fdfac16a58523a866ddb68d/bc4d30a9-a33b-4308-9649-e922fb5f7413)

**Comment by MoeNick** [comment]
@laurenluz @markoprljic 
I also add a comment like this in our new design Figma. It might be understandable if we want to have less drop-down there, but we have to confirm the logic that If "Verified" or "Trace" is selected, we can move "verified / trace"  at the top but newest first? or shall we add sort bey verified-newest...

It's all UX I think, technically we can have every query needed if design confirms how the look should be.

**Comment by laurenluz** [comment]
Yeah maybe having an extra drop down is not the best solution, but the way it is now is confusing for the user.

we can do "'sort by' verified-newest" instead... that makes sense to me

**Comment by mohammadranjbarz** [comment]
@laurenluz  @MoeNick 
Through Giveth/giveth-next#319 we had some changes that it's related to this issue
please check https://next.giveth.io and Giveth/giveth-next#319

**Comment by mateodaza** [comment]
I'm sure new design will take this into consideration, so no need to focus now on this? @MoeNick @laurenluz what do you think?

**Comment by MoeNick** [comment]
I think what we have already on staging is fine both on next and typescript.
